### PR TITLE
682 configuration resources API in site settings module

### PR DESCRIPTION
**Pull request by hwindsor**
## Pull Request Overview

- Moved Configuration resources into Site settings
- All apis unchanged. This is the first step on the path of refactoring the configuration

---

### CODE REVIEW

File: app/modules/site_settings/models.py [hunk 1]
```diff
@@ -21,6 +26,16 @@ class SiteSetting(db.Model, Timestamp):
         'confirm_deleted_rows': False,
     }
 
+    HOUSTON_SETTINGS = {
```
Comment (hwindsor) [inline app/modules/site_settings/models.py:29]: Jon, this was in place of all the special cases for certain variables through the code. This seemed much more scalable.

File: app/modules/site_settings/parameters.py [hunk 2]
```diff
@@ -9,9 +9,9 @@
 from . import schemas
 
 
-class CreateSiteSettingParameters(Parameters, schemas.DetailedSiteSettingSchema):
-    class Meta(schemas.DetailedSiteSettingSchema.Meta):
-        fields = schemas.DetailedSiteSettingSchema.Meta.fields + (
+class CreateSiteSettingFileParameters(Parameters, schemas.DetailedSiteSettingFileSchema):
```
Comment (hwindsor) [inline app/modules/site_settings/parameters.py:12]: What was site settings API will become a file only API, names changed to reflect this

**Review by colinwkingen — APPROVED** [review]
LGTM

**Comment by naknomum** [comment]
is it me, or is this leaving behind an empty `app/modules/configuration/` directory?   is this some git thing that it cant be outright removed cuz of the pycache dir in it or something?

**Review by naknomum — APPROVED** [review]
looks good.  ran it locally and did a bit of testing and seemed to be backwards-compatible etc.  :+1:

**Pull request merged**